index.bs

```diff
@@ -73,6 +76,8 @@ interface CrashReportBody : ReportBody {
   [Default] object toJSON();
   readonly attribute DOMString? reason;
   readonly attribute DOMString? stack;
+  readonly attribute DOMString? is_top_level;
+  readonly attribute DocumentVisibilityState? page_visibility;
```

Comment:
Out of curiosity, what’s the reason these are using `snake_case`? In JavaScript APIs, `camelCase` seems to be more common. Why not `isTopLevel` / `pageVisibility`?

Reply:
Good question. Snake casing here follows the TAG's guidance in https://w3ctag.github.io/design-principles/#casing-rules for JSON keys. @clelland also pointed me to https://github.com/w3ctag/design-reviews/issues/585#issuecomment-768563886 which records this decision being made elsewhere in the reporting world. Hope that helps clear it up!